### DISCUSSION TITLE
"Welcome to MonitorControl" window shows an "Open System Preferences..." button, which does not launch System Settings in Ventura.

### DISCUSSION THREAD

**Discussion by bradtchapman** [MonitorControl/MonitorControl#1406] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

On macOS Ventura 13.4, the first time / welcome window shows a "Open System Preferences" button, but does not launch the System Settings app as expected.  

I had to open System Settings manually to visit the Preference Pane and grant accessibility to MonitorControl. 

### Steps to reproduce

1. Install MonitorControl 4.1.0 on a Mac running latest Ventura build.
2. If this is a reinstall, ensure all preferences are cleared out, as well as the ~/Library/Containers/MonitorControlHelper folder.
3. Launch MonitorControl and ensure you see the Welcome screen.
4. Click the "Open System Preferences..." button.
5. Nothing happens.
6. Verify that System Settings app is not currently running.
7. Keep clicking the button anyway in vain.
8. Give up and follow the directions to manually add MonitorControl in Privacy & Security.

### Expected behavior

The button in the Welcome pane should have opened System Settings to the desired pane.

### Anything else?

![Screenshot 2023-06-17 at 9 24 20 PM](https://github.com/MonitorControl/MonitorControl/assets/32527656/8c3cb2c0-f94c-44d6-8209-a8976ae4e808)

### Environment Information (please complete the following information)

```markdown
- macOS version: 13.4
- Mac model: M2 MacBook Pro
- MonitorControl version: 4.1.0
- Monitor(s): Dell U2720Q, Dell P2422H
- Apple Silicon/M1 (yes or no): Yes
```

**Comment by waydabber** [comment]
I think that button is not meant to opten System Preferences but initiate the OS's built in flow to configure accessibility security settings. However if these are already configured, nothing will happen.
